lis/surfacemodels/land/noahmp.3.6/module_sf_noahmplsm_36.F90

```diff
@@ -7268,8 +7268,8 @@ SUBROUTINE SNOWWATER (NSNOW  ,NSOIL  ,IMELT  ,DT     ,ZSOIL  , & !in
    enddo
 
 !to obtain equilibrium state of snow in glacier region
-       
-   IF(SNEQV > 2000.) THEN   ! 2000 mm -> maximum water depth
+! Hans Lievens added (DZSNSO(0) > 0):       
+   IF(SNEQV > 2000. .AND. DZSNSO(0) > 0) THEN   ! 2000 mm -> maximum water depth
```

Comment:
Could you clarify why here it is DZSNSO(0) > 0 and in noahmp36_snow_update.F90 it's DZSNSO(0) > 0.0001.

Reply:
I think in the old LIS version  DZSNSO(0) > 0.0001 wasn't present yet. It seems like we made a similar adjustment as other LIS users. I think both options are fine.

Reply:
But both adjustments of the "if statement" were made by Hans. If possible I suggest to harmonize and perhaps add a comment why this threshold is needed.

Reply:
now harmonized by Isis